2-Draft-Accepted/RFCNNNN-Support-Experimental-Features.md

```diff
@@ -457,6 +470,27 @@ as well as a problem that we have to solve before using the experimental feature
 
 > Not sure how to handle it yet ...
```

Comment:
For testing, I think we can simplify this.  Due to the nature of being experimental, I think our regular test passes should run without any experimental features enabled.  Separately, we should run the experimental feature tests with all experimental features enabled and all of the CI/Feature tests.

Reply:
I'm not sure that this will work. I easily envision conflicting or mutually exclusive experimental features as mentioned. I wonder if we are going to need more tags.

Reply:
I agree our regular test pass should run without any experimental feature enabled.

For the experimental features, we need to run test passes for each of them (one per test run) enabled. We will need more tags because tests targeting an experimental feature should be skipped sometimes.